inspektor/utils/download.py

```diff
@@ -18,6 +18,8 @@
 import socket
 import six
 
+# pylint: disable=E0611
+# pylint: disable=E1101
```

Comment:
It'd be nice to split the commit into style fixes and the functional changes (travis/setup.py/...)